packages/web/src/components/collection/mobile/CollectionHeader.tsx

```diff
@@ -273,10 +273,18 @@ const CollectionHeader = ({
           onRepost={onRepost}
           onClickOverflow={onClickOverflow}
           onClickEdit={handleClickEdit}
-          showFavorite={!!onSave && !isOwner && hasStreamAccess}
-          showRepost={variant !== Variant.SMART && !isOwner && hasStreamAccess}
-          showShare={variant !== Variant.SMART || type === 'Audio NFT Playlist'}
-          showOverflow={variant !== Variant.SMART}
+          showFavorite={!!onSave && !isOwner && hasStreamAccess && !isPrivate}
+          showRepost={
+            variant !== Variant.SMART &&
+            !isOwner &&
+            hasStreamAccess &&
+            !isPrivate
+          }
+          showShare={
+            (variant !== Variant.SMART || type === 'Audio NFT Playlist') &&
+            !isPrivate
+          }
+          showOverflow={variant !== Variant.SMART && !isPrivate}
```

Comment:
not for today, but I wish we could just push this logic down to the menu/buttons themselves